forChanna.scd

```diff
@@ -22,6 +20,7 @@
         ~player = (~libPath ++ "/controllers/player.scd").load;
         ~player.create(~score, ~playlist, 0.1, ~colorMaker);
 
+        s.bootSync;
```

Comment:
change to .reboot

Reply:
I'm okay with rebooting (though since we're talking about multiple concurrent realizations as a goal, we may eventually have to consider the problem of instantiating a new realization while an existing one is playing), but by itself reboot doesn't sync so this makes it harder for us to reason about the state of the Server.

Here are two ways of combining `reboot` and `sync`, the first more explicit, the second more concise:
```
(
    {
        // do everything that doesn't depend on server state here

        // block while server reboots
        ~serverRunning = Condition.new;
        s.reboot;
        s.doWhenBooted({
            s.sync; 
            ~serverRunning.test_(true);
            ~serverRunning.signal
        });
        ~serverRunning.wait;

        // do everything that depends on server state here

        "Config OK".postln
    }.fork
)
```
```
// you can also do this without Condition...
(
    s.reboot;
    s.doWhenBooted({
        {
            s.sync;
            // ...but if we added anything to setup that depended on server state it would have to go in this Function.
            "Config OK".postln
        }.fork
    })
)
```
I'd prefer the first. What do you think?

Reply:
no real dog in this fight, let's go with first.

Reply:
Sounds good; just pushed changes based on first option plus some additional insights from our discussion in #7. Let me know what you think!